manifest.json

```
@@ -267,6 +267,26 @@
               "fedora-32-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
             }
           }]
+        },
+        {
+          "platforms": [{
+            "dockerfile": "src/fedora/33/amd64",
+            "os": "linux",
+            "osVersion": "fedora33",
+            "tags": {
+              "fedora-33-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
+            }
```

You're going to need a local tag.  By default, tags in this repo are configured to published with timestamps in the tag name.  But your Helix Dockerfile references a tag that doesn't have the timestamp in it, because it can't know what it is.  For that reason, we use things called local tags which allow you to define the tag locally but not actually publish it.  To configure that, you can define it like this.

```suggestion
              "fedora-33-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
              "fedora-33": {
                "isLocal": true
              }
            }
```